Cargo.toml

```diff
@@ -50,9 +50,10 @@ serde_urlencoded = "0.5"
 comrak = "0.4"
 serde_json = "1.0"
 serde_derive = "1.0"
-diff-rs = "0.2"
-patch-rs = "0.5"
-splitdiff-rs = "0.4"
+diff-rs = "0.2.3"
+patch-rs = "0.6.2"
+splitdiff-rs = "0.4.1"
+interdiff-rs = "0.2.0"
```

Comment:
only major version num in Cargo.toml
so that minor versions can be updated without extra effort

Reply:
done